src/sgd/frontend/yeastgenome/static/templates/snapshot.jinja2

```diff
@@ -64,7 +64,7 @@
 				<h2>Summary of Phenotype Annotations<span class="date-container"></span></h2>
 				<hr />
 				<p>
-					This bar graph represents the state of phenotype annotations for the entire <i>S. cerevisiae</i> genome using a phenotype slim (i.e., a high-level subset of phenotype terms). This subset allows phenotypes to be grouped into broad categories. More information about phenotype annotation and the ontology can be found on SGD’s <a target="_blank" href="https://sites.google.com/view/yeastgenome-help/function-help/phenotypes">phenotype help page</a>. All the phenotype data summarized on this graph can be found in our <a href="http://www.yeastgenome.org/downloads/curation">phenotype_data.tab</a> file.
+					This bar graph represents the state of phenotype annotations for the entire <i>S. cerevisiae</i> genome using a phenotype slim (i.e., a high-level subset of phenotype terms). This subset allows phenotypes to be grouped into broad categories. More information about phenotype annotation and the ontology can be found on SGD’s <a target="_blank" href="https://sites.google.com/view/yeastgenome-help/function-help/phenotypes">phenotype help page</a>. All the phenotype data summarized on this graph can be found in our <a href="https://downloads.yeastgenome.org/curation/literature/">phenotype_data.tab</a> file.
```

Comment:
let's change this URL to /search?q=phenotype_data&category=download&status=Active